tests/chainerx_tests/unit_tests/routines_tests/test_normalization.py

```diff
@@ -1,10 +1,11 @@
-import chainer
 import numpy
 import pytest
 
+import chainer
```

Comment:
It seems it should be at the original place?

Reply:
Fixed.